app/controllers/facilities_controller.rb

```diff
@@ -3,4 +3,11 @@
 
 class FacilitiesController < ApplicationController
   skip_before_action :authenticate
+
+  def validate_bbox
+    raise ArgumentError unless params[:bbox].length == 4
+    params[:bbox].each { |x| Float(x) }
```

Comment:
The ArcGIS query API is pretty tolerant - you can put values out of this range in, and it will coerce them into reasonable lat-long boundaries. So from the perspective of guarding the underlying API, there's not a need. But there's some value in protecting our API in case we use some non ArcGIS geo-backend sometime.

Also it made me realize that this validation would accept inputs like "0x1f", whereas ArcGIS rejects hex/other base values, so need to coerce to or validate as decimal inputs.